pbj-core/pbj-runtime/src/test/java/com/hedera/pbj/runtime/ProtoWriterToolsTest.java

```diff
@@ -280,19 +310,44 @@ void testWriteBoolean_false() {
         assertEquals(0, bufferedData.length());
     }
 
+    private static EnumWithProtoMetadata mockEnum(int ordinal) {
+        EnumWithProtoMetadata enumWithProtoMetadata = mock(EnumWithProtoMetadata.class);
+        doReturn(ordinal).when(enumWithProtoMetadata).protoOrdinal();
+        doReturn(RANDOM_STRING.nextString()).when(enumWithProtoMetadata).protoName();
+        return enumWithProtoMetadata;
+    }
+
     @Test
     void testWriteEnum() {
         FieldDefinition definition = createFieldDefinition(ENUM);
-        EnumWithProtoMetadata enumWithProtoMetadata = mock(EnumWithProtoMetadata.class);
-        int expectedOrdinal = RNG.nextInt();
-        when(enumWithProtoMetadata.protoOrdinal()).thenReturn(expectedOrdinal);
-        when(enumWithProtoMetadata.protoName()).thenReturn(RANDOM_STRING.nextString());
+        final int expectedOrdinal;
+        EnumWithProtoMetadata enumWithProtoMetadata = mockEnum(expectedOrdinal = RNG.nextInt());
```

Comment:
I wonder why `expectedOrdinal` is initialized here rather than on the previous line?

Reply:
No particular reason. Moved the assignment to the declaration.